alibi_detect/od/mahalanobis.py

```diff
@@ -63,10 +63,10 @@ def __init__(self,
         # keys = categorical columns; values = numerical value for each of the categories
         self.cat_vars = cat_vars
         self.ohe = ohe
-        self.d_abs = {}  # type: Dict
+        self.d_abs: Dict = {}
 
         # initial parameter values
-        self.clip = None  # type: Union[None, list]
+        self.clip: Union[None, list] = None
```

Comment:
Use the opportunity to convert to `Optional[list]`?

Reply:
Done in [9c910ac](https://github.com/SeldonIO/alibi-detect/pull/725/commits/9c910ac80265f21097499c6c3207cb12b70997af)